### PR TITLE
Remove "Still Empty" Sections

### LINKED ISSUE #520: Remove "Still Empty" Sections
## Feature Description

There are sections we are "alpha" launching without and need to remove those sections from the UI.

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria
1. Remove Element Panel - Elements section
2. Remove Element Panel - Animations section
3. Remove Prepublish list (Ensure stories are published after hitting the publish button)
4. Remove search bar from shapes tab on element panel (Pending Sam's confirmation)
5. Remove search bar from text tab on element panel (Pending Sam's confirmation)

### PR DESCRIPTION
**Pull request by merapi**
Fixes #520 
Waiting for @samitron7  UX confirmation.

### CODE REVIEW

File: assets/src/edit-story/components/library/panes/shapes/shapesPane.js [hunk 1]
```diff
@@ -63,11 +63,6 @@ function ShapesPane(props) {
       <Header>
         <Title>{__('Shapes', 'web-stories')}</Title>
       </Header>
-      <SearchInput
```
Comment (dvoytenko) [inline assets/src/edit-story/components/library/panes/shapes/shapesPane.js]: Seems like both shapes and text panel supposed to support search input per figma?

Comment (merapi) [reply assets/src/edit-story/components/library/panes/shapes/shapesPane.js]: Correct, but probably not now (currently non-functional, so removed) https://github.com/google/web-stories-wp/issues/520#issuecomment-598379439

Comment (dvoytenko) [reply assets/src/edit-story/components/library/panes/shapes/shapesPane.js]: Ok. Let's just make sure we file a bug if removing an expected feature.

**Review by dvoytenko — APPROVED** [review]

**Comment by pbakaus** [comment]
@samitron7 I'm arguing we should keep the animation tab and show some "coming soon" message, but merge the elements and shape panel into one (and keep the icon of the shape panel).

I'd like to merge this soon so we can follow up with https://github.com/google/web-stories-wp/pull/657.

@merapi let's not kill the Search boxes yet, as I want to make sure the styling is sound in the other PR first.

**Comment by merapi** [comment]
Applied latest comments from Slack: _remove the element panel, but keep the animations panel_

**Pull request merged**